_integration-schemas/stripe/v1/payout_transactions.md

```diff
@@ -0,0 +1,27 @@
+---
+tap: "stripe"
+version: "1"
+
+name: "payout_transactions"
+doc-link: "https://stripe.com/docs/api/payouts"
+singer-schema: "https://github.com/singer-io/tap-stripe/blob/master/tap_stripe/schemas/payout_transactions.json"
+description: "This table contains info about payout transactions."
+
+replication-method: "Full Table Replication"
```

Comment:
When the `replication-method` is full, it should just be `Full Table`. No need for `Replication` to be in here. I fixed it, but just lettin' ya know.